client/assets/stylesheets/shared/mixins/_core-typography.scss

```diff
@@ -0,0 +1,102 @@
+// ==========================================================================
+// Core Typography definitions
+//
+// See: https://a8cforagencies.wordpress.com/2025/01/16/request-for-comments-using-core-typography/
```

Comment:
Let's not link to internal links.

```suggestion
// See: pfunGA-4Ch-p2
```

Reply:
Good point, I have removed it from here.